.github/workflows/e2e.yaml

```diff
@@ -101,6 +102,9 @@ jobs:
           k8s_version: ${{ inputs.k8s_version }}
           ip_family: ${{ inputs.suite == 'IPv6' && 'IPv6' || 'IPv4' }} # Set the value to IPv6 if IPv6 suite, else IPv4
           git_ref: ${{ inputs.git_ref }}
+      - name: cloudformation describe stack events
+        if: ${{ failure() && steps.create-eks-cluster.outcome != 'success' }}
+        run: aws cloudformation describe-stack-events --stack-name=iam-${{ env.CLUSTER_NAME }}
```

Comment:
I believe there are many stacks that are spawned by the `eksctl` command. Did we make sure that we handle describe stack events if _any_ of these different stacks fails, not just the IAM stack.

Reply:
Maybe we can list stacks by tag and then describe stack events across all stacks on failure?

Reply:
Updated this